.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "rust-analyzer.rustfmt.extraArgs": ["+nightly"]
+}
```

Comment:
I personally don't use VSCode (and I see it as a personal choice which editor/IDE you use), can we remove this?

Reply:
I mainly added it as in vscode it will hard error on format as by default it uses stable